### PR TITLE
bug fixes from content

### PR DESCRIPTION
**Pull request by kave**
@kurtw 
@KimberlyMunoz

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/molecules/text-introduction.html [hunk 1]
```diff
@@ -34,7 +34,12 @@ <h1 class='m-text-introduction_heading'>{{ value.heading }}</h1>
         <p class='m-text-introduction_body'>{{ value.body | safe }}</p>
 
         <div class='m-text-introduction_link-container'>
-            <a class='jump-link' href='{{ value.link.url }}'>{{ value.link.text }}</a>
+            {% for link in value.links %}
+                <li class="list_item">
+                    <a class='jump-link' href='{{ link.url }}'>{{ link.text }}</a>
+                </li>
+            {% endfor %}
+
```
Comment (kave) [inline cfgov/jinja2/v1/_includes/molecules/text-introduction.html]: @KimberlyMunoz hey is this good enough for now a list of links in text introduction?

Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/molecules/text-introduction.html]: Yep. I think so.

File: cfgov/cfgov/settings/base.py [hunk 2]
```diff
@@ -265,9 +265,10 @@
 if os.environ.get('S3_ENABLED', False):
     DEFAULT_FILE_STORAGE = 'v1.s3utils.MediaRootS3BotoStorage'
     AWS_S3_SECURE_URLS = False  # True = use https; False = use http
-    AWS_QUERYSTRING_AUTH = False  # don't add complex authentication-related query parameters for requests
+    AWS_QUERYSTRING_AUTH = False  # False = do not use authentication-related query parameters for requests
     AWS_S3_ACCESS_KEY_ID = os.environ.get('AWS_S3_ACCESS_KEY_ID')
     AWS_S3_SECRET_ACCESS_KEY = os.environ.get('AWS_S3_SECRET_ACCESS_KEY')
     AWS_STORAGE_BUCKET_NAME = os.environ.get('AWS_STORAGE_BUCKET_NAME')
+    AWS_S3_CALLING_FORMAT = 'boto.s3.connection.OrdinaryCallingFormat'
```
Comment (kave) [inline cfgov/cfgov/settings/base.py]: allows use of dot notation bucket names in requests

**Comment by kurtw** [comment]
looks fine to me :+1:

**Pull request merged**